<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>930</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1900</width>
    <height>930</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1900</width>
    <height>930</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>USE Calculator</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>0</y>
     <width>4</width>
     <height>930</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonSearch">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>870</y>
     <width>370</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="L9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>780</y>
     <width>261</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Litetature</string>
   </property>
  </widget>
  <widget class="QLabel" name="L8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>720</y>
     <width>261</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Social Science</string>
   </property>
  </widget>
  <widget class="QLabel" name="L7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>660</y>
     <width>261</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>History</string>
   </property>
  </widget>
  <widget class="QLabel" name="L6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>600</y>
     <width>261</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Foreign Language</string>
   </property>
  </widget>
  <widget class="QLabel" name="L5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>280</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Geography</string>
   </property>
  </widget>
  <widget class="QLabel" name="L4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>261</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Biology</string>
   </property>
  </widget>
  <widget class="QLabel" name="L3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>261</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Chemistry</string>
   </property>
  </widget>
  <widget class="QLabel" name="L2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>261</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Physics</string>
   </property>
  </widget>
  <widget class="QLabel" name="L1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>261</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Informatics</string>
   </property>
  </widget>
  <widget class="QLabel" name="MAT">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>261</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Mathematics</string>
   </property>
  </widget>
  <widget class="QLabel" name="RUS">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>261</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Russian Language</string>
   </property>
  </widget>
  <widget class="QLabel" name="Hello">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>371</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter your exam points  and press &quot;Search&quot;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="BRus">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="BMat">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>240</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="B1">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="B2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>360</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="B3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>420</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="B4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>480</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="B5">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>540</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="B6">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>600</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="B7">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>660</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="B8">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>720</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="B9">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>780</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="Message">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>469</x>
     <y>34</y>
     <width>1291</width>
     <height>891</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Evaluate your admission chances!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="AddButton">
   <property name="geometry">
    <rect>
     <x>774</x>
     <y>870</y>
     <width>741</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add new table record</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>19</y>
     <width>1431</width>
     <height>831</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <property name="columnCount">
    <number>7</number>
   </property>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QLabel" name="Sum">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>870</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="Form">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>1430</width>
     <height>840</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Fill the information about educational program</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>400</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>University</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>158</y>
      <width>400</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>Faculty</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>256</y>
      <width>400</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>Specialty</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>354</y>
      <width>400</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>Extra  exams</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>550</y>
      <width>400</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>Entrance points</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>648</y>
      <width>400</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>Degree</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>746</y>
      <width>400</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>Additional exam</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="Name">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>60</y>
      <width>910</width>
      <height>60</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="Faculty">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>158</y>
      <width>910</width>
      <height>60</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="Specialty">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>256</y>
      <width>910</width>
      <height>60</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="ExtEx1">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>354</y>
      <width>910</width>
      <height>60</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="ExtEx2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>452</y>
      <width>910</width>
      <height>60</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="Score">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>550</y>
      <width>141</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QTextEdit" name="Degree">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>648</y>
      <width>910</width>
      <height>60</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="AddEx">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>746</y>
      <width>910</width>
      <height>60</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="SaveButton">
   <property name="geometry">
    <rect>
     <x>774</x>
     <y>870</y>
     <width>741</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <zorder>Message</zorder>
  <zorder>line</zorder>
  <zorder>ButtonSearch</zorder>
  <zorder>L9</zorder>
  <zorder>L8</zorder>
  <zorder>L7</zorder>
  <zorder>L6</zorder>
  <zorder>L5</zorder>
  <zorder>L4</zorder>
  <zorder>L3</zorder>
  <zorder>L2</zorder>
  <zorder>L1</zorder>
  <zorder>MAT</zorder>
  <zorder>RUS</zorder>
  <zorder>Hello</zorder>
  <zorder>BRus</zorder>
  <zorder>BMat</zorder>
  <zorder>B1</zorder>
  <zorder>B2</zorder>
  <zorder>B3</zorder>
  <zorder>B4</zorder>
  <zorder>B5</zorder>
  <zorder>B6</zorder>
  <zorder>B7</zorder>
  <zorder>B8</zorder>
  <zorder>B9</zorder>
  <zorder>AddButton</zorder>
  <zorder>tableWidget</zorder>
  <zorder>Sum</zorder>
  <zorder>Form</zorder>
  <zorder>SaveButton</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
